src/Controls/src/Core/PlatformPointerEventArgs.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+/// <summary>
+/// Platform-specific arguments associated with the PointerEventArgs
+/// </summary>
+public class PlatformPointerEventArgs
+{
+#if IOS || MACCATALYST
+	/// <summary>
+	/// UIView that has the attached PointerGestureRecognizer
+	/// </summary>
+	public UIKit.UIView Sender { get; }
+
+	/// <summary>
+	/// UIGestureRecognizer attached to the UIView
+	/// </summary>
+	public UIKit.UIGestureRecognizer GestureRecognizer { get; }
+
+	internal PlatformPointerEventArgs(UIKit.UIView sender, UIKit.UIGestureRecognizer gestureRecognizer)
+	{
+		Sender = sender;
+		GestureRecognizer = gestureRecognizer;
+	}
+
+#elif ANDROID
+	/// <summary>
+	/// View that has the attached MotionEvent
+	/// </summary>
+	public Android.Views.View Sender { get; }
+
+	/// <summary>
+	/// MotionEvent attached to the View
+	/// </summary>
+	public Android.Views.MotionEvent MotionEvent { get; }
+
+	internal PlatformPointerEventArgs(Android.Views.View sender, Android.Views.MotionEvent motionEvent)
+	{
+		Sender = sender;
+		MotionEvent = motionEvent;
+	}
+
+#elif WINDOWS
+	/// <summary>
+	/// FrameworkElement that has the attached RoutedEventArgs
```

Comment:
```suggestion
	/// Gets the <see cref="FrameworkElement/>` that has the attached <see cref=`RoutedEventArgs`/>.
```
You might need to double check that intellisense can catch on the cref for FrameElement. Otherwise you'll need to put in the full namespace.

https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property